pdc/apps/common/filters.py

```diff
@@ -50,6 +52,23 @@ def filter(self, qs, value):
         return qs
 
 
+class MultiValueRegexFilter(MultiValueFilter):
```

Comment:
It seems the query operation in DRF supports &(And), will this change it to OR?
Why we want to support multi value regex?